src/io/github/sspanak/tt9/languages/definitions/Polish.java

```diff
@@ -0,0 +1,22 @@
+package io.github.sspanak.tt9.languages.definitions;
+
+import java.util.Arrays;
+import java.util.Locale;
+
+public class Polish extends English {
+	public Polish() {
+		super();
+
+		locale = Locale.POLISH;
+		dictionaryFile = "pl-utf8.txt";
+
+		isPunctuationPartOfWords = true;
```

Comment:
This must be off. Polish words don't contain apostrophes like "don't".

```suggestion
```

Reply:
Do compound words count? E.g. "czarno-biały" means "black and white" and includes a dash.

Reply:
There are no such words in the word list, so there is no need to enable the option.

Either way, it was a hack for Dutch language, where there are words that start with an apostrophe. It wasn't a great idea, but at the time, I couldn't come up with anything better. But now, I am planning to get rid of it, so don't worry too much about it.